core/include/core/pybindings.h

```diff
@@ -174,7 +173,10 @@ class G3ModuleRegistrator {
 //     SPT3G_PYTHON_MODULE(foo)
 // for a package whose name will be _libfoo
 #define SPT3G_PYTHON_MODULE(name) \
-BOOST_PYTHON_MODULE(BOOST_PP_CAT(_lib, name))
+BOOST_PYTHON_MODULE(_lib ## name)
```

Comment:
Same issue as with `BOOST_PP_STRINGIZE`, but likely less concerning, as probably no one wants to generate a module name from a macro.

Reply:
Agreed, I'm not super worried about this piece, so I'll leave it as is here.  If we start making more complicated module names, I'm sure this whole `SPT3G_PYTHON_MODULE()` macro will change all over again.